### DISCUSSION TITLE
Remove the fork relationship to make the code searchable

### DISCUSSION THREAD

**Discussion by AlphaJack** [jonaswinkler/paperless-ng#891] [Feature Requests]
I am encountering the following error when searching in the code:

> Sorry, forked repositories are not currently searchable.

Would you consider removing the fork status anytime soon? AndOTP had a [similar discussion](https://github.com/andOTP/andOTP/issues/145)

**Comment by jonaswinkler** [comment]
I've already thought about that. I Should probably just do it.

**Reply by jgillula** [reply]
That would be amazing. I'd like to contribute to paperless-ng and that would make it so much easier.

**Reply by colinfrei** [reply]
I just came across this limitation as well, would be great to have the repository be searchable 👍

**Reply by jonaswinkler** [reply]
I've sent a ticket to GitHub support!

**Reply by jonaswinkler** [reply]
Done.
